packages/meteor/dynamics_nodejs.js

```diff
@@ -2,6 +2,18 @@
 let nextSlot = 0;
 let callAsyncMethodRunning = false;
 
+var Fiber = Npm.require('fibers');
```

Comment:
Why is this code back?

This import will always break because the fibers module is not installed anymore.